javascript/schema/src/codebook/ColumnError.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+export interface ColumnError {
+	columnName: string
```

Comment:
You can just call this `name` for simplicity

Reply:
You might even consider naming the interface `FieldError` or `ValidationError` to make it more reusable. One of the reasons I used `Field` in the codebook was because it may not always be a column in a table - it could be a property on a graph node for example.